.github/workflows/ci.yml

```
@@ -635,8 +635,11 @@ jobs:
             bash -euo pipefail -c \
             "./build_tools/cmake/build_riscv.sh && tests/riscv32/smoke.sh"
 
+  # TODO(#10391): Update the benchmark dependency with the new benchmark
+  # framework. The step only requires the compiled benchmark modules, not the
+  # not the result from the benchmark.
```

typo: "not the not the"

```suggestion
  # TODO(#10391): Update the benchmark dependency with the new benchmark
  # framework. The step only requires the compiled benchmark modules, not the
  # result from the benchmark.
```

---

Oops. Sorry. Fixed.